src/extensions/default/bramble/lib/Theme.js

```diff
@@ -57,7 +57,7 @@ define(function (require, exports, module) {
         ThemeManager.addTheme(lightFile, lightOptions);
         ThemeManager.addTheme(darkFile, darkOptions);
 
-        currentTheme = theme === 'light-theme' ? "dark-theme" : "light-theme";
+        currentTheme = theme === 'light-theme' ? "dark-theme" : "dark-theme";
```

Comment:
I'm not sure what this code was meant to do, but it's also wrong.  We should just take `theme` as passed and use that, or stick with whatever `currentTheme` is.  I'd say this code got messed up in the recent past, which might explain why it doesn't retain the choice with a refresh.